build.gradle.kts

```diff
@@ -25,15 +25,16 @@ dependencies {
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.0")
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactor:1.6.0")
 
-  implementation("org.flywaydb:flyway-core:8.4.4")
+  implementation("org.flywaydb:flyway-core:8.5.1")
   implementation("com.vladmihalcea:hibernate-types-52:2.14.0")
   runtimeOnly("io.r2dbc:r2dbc-postgresql")
   runtimeOnly("org.springframework.boot:spring-boot-starter-jdbc")
-  runtimeOnly("org.postgresql:postgresql:42.3.2")
+  runtimeOnly("org.postgresql:postgresql:42.3.3")
   implementation("io.opentelemetry:opentelemetry-api:1.11.0")
 
   implementation("org.springdoc:springdoc-openapi-webflux-ui:1.6.6")
   implementation("org.springdoc:springdoc-openapi-kotlin:1.6.6")
+  implementation("org.springdoc:springdoc-openapi-security:1.6.6")
```

Comment:
not really needed, but if using `AuthenticationPrincipal` then will automatically ignore